src/data-interfaces/HoloFuelDnaInterface.js

```diff
@@ -12,8 +12,20 @@ const createZomeCall = instanceCreateZomeCall(INSTANCE_ID)
 
 const MOCK_DEADLINE = '4019-01-02T03:04:05.678901234+00:00'
 
+/* Creates an array of all transactions that inlcude new counterparties for a provided transaction list and counterparty list */
+export const findnewCounterpartiesFromList = (transactionList = [], counterpartyList = []) => {
+  // eslint-disable-next-line array-callback-return
+  const newCouterpartyList = transactionList.filter(({ counterparty }) => {
+    if (isEmpty(counterpartyList)) return counterparty
```

Comment:
because this function is being passed to filter, it should only ever return `true` and `false` (or other falsey values). It's confusing to return `counterparty` when we're not using the value, just the truthiness of it.